### PR TITLE
Add email subscription

### LINKED ISSUE #3: Listen to lp-prod-reconciliation-response and send email notification
Add a listener to the lp-prod-reconciliation-response SNS topic and forward messages to hls-impact@uah.edu

### PR DESCRIPTION
**Pull request by chuckwondo**
Fixes #3

### CODE REVIEW

**Review by ceholden — APPROVED** [review]
+1 this should address the issue ✅. Thanks for the detail in the README about what to expect for the emails from the integration test 💯

Only thought I had was that you might end up getting emails from other people's integration tests 📬. It doesn't look like we have a PR template but that might be a nice place to remind "Remidner to update the `HLS_LPDAAC_NOTIFICATION_EMAIL_ADDRESS` Github environment variable for `dev` if you'd like to see the email from integration test"

**Comment by chuckwondo** [comment]
> +1 this should address the issue ✅. Thanks for the detail in the README about what to expect for the emails from the integration test 💯
> 
> Only thought I had was that you might end up getting emails from other people's integration tests 📬. It doesn't look like we have a PR template but that might be a nice place to remind "Remidner to update the `HLS_LPDAAC_NOTIFICATION_EMAIL_ADDRESS` Github environment variable for `dev` if you'd like to see the email from integration test"

Yes, I've been thinking about how to improve this, because I've currently set it to my devseed email in the repo's GitHub dev env just for expediency, but I'm wondering if we can grab the PR author's email in the context of the github workflow, and set that as the value of the env var on-the-fly.

**Comment by chuckwondo** [comment]
Hey @ceholden. I sorted out the email configuration such that it grabs the email address of the author of the PR, and I eliminated the corresponding env var from the `dev` environment. It's only configured in the `prod` environment now. Just want you to take another look to see the latest changes.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -131,16 +139,29 @@ jobs:
...
             jq -r <<<"$VARS_JSON" 'to_entries|map("\(.key)=\(.value)\u0000")[]'
           )
-      - name: Deploy integration test stack
-        run: |
-          make deploy-it
+      - name: Get user's email address
+        id: get-email-address
+        uses: actions/github-script@v7
+        with:
+          script: |
```
Comment (ceholden) [inline .github/workflows/main.yml:146]: This is awesome!

I was thinking if we would want to mask this output but it looks like Github will do the `@users.noreply.github.com` if your Github settings has the email visibility set to "private", so ✅

**Review by ceholden — APPROVED** [review]
💯 great work making the integration tests for the email notification "just work" for everyone who develops this in the future

**Pull request merged**